setup.py

```diff
@@ -0,0 +1,53 @@
+import sys
+from setuptools import setup, find_packages, Extension
+from Cython.Distutils import build_ext
+from Cython.Build import cythonize
+
+ext_modules = []
+
+names = [
+    'pyspades.vxl',
+    'pyspades.bytes',
+    'pyspades.packet',
+    'pyspades.contained',
+    'pyspades.common',
+    'pyspades.world',
+    'pyspades.loaders',
+    'pyspades.mapmaker'
+]
+
+for name in names:
+    extra = {'extra_compile_args' : ['-std=c++11']} if name in ['pyspades.vxl', 'pyspades.world', 'pyspades.mapmaker'] else {}
+
+    ext_modules.append(Extension(name, ['%s.pyx' % name.replace('.', '/')],
+        language = 'c++', include_dirs=['pyspades'], **extra))
+
+
+setup(
+    name = 'pysnip',
+    packages = ['pysnip', 'pysnip.web', 'pyspades', 'pyspades.enet'],
+    version = '0.0.0',
+    description = 'Open-source server implementation for Ace of Spades',
+    author = 'Matpow2, Stackoverflow',
+    author_email = 'nate.shoffner@gmail.com',
+    url = 'https://github.com/NateShoffner/PySnip',
+    download_url = 'https://github.com/NateShoffner/PySnip/archive/master.tar.gz',
```

Comment:
might want to update this info for spades-server - including the `name` to `spades-server`. Obviously keep info about original authors and copyright, but probably should go in the readme.